student.rb

```diff
@@ -0,0 +1,17 @@
+module StudentModule
+  require_relative 'person'
+  include PersonModule
```

Comment:
- [ ] Hello and good job on adding the classes, but the `include PersonModule` I don't know what purpose.